src/libs/actions/BankAccounts.js

```diff
@@ -553,7 +553,6 @@ function validateBankAccount(bankAccountID, validateCode) {
             if (response.jsonCode === 200) {
                 Growl.show('Bank Account successfully validated!', CONST.GROWL.SUCCESS, 3000);
                 Navigation.dismissModal();
-                return;
```

Comment:
Nice catch! 

But I think the intention to early return here is correct as the non 200 case would be error handling logic. 

So we just need to set loading to `false` and `error` to empty string maybe `''`?

```js
    Navigation.dismissModal();
    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: false, error: ''});
    return;
}
```